src/iotile/models/datablock.ts

```diff
@@ -17,5 +18,14 @@ export class DataBlock {
     this.sensorGraphSlug = data.sg;
     this.createdOn = new Date(data.created_on);
     this.createdBy = data.created_by;
+    this.description = data.description;
+  }
+
+  public getPostPayload() {
+    return {
+      title: this.title,
+      description: this.description,
```

Comment:
This should be protected with an `if this.description` as there may not be any description.